packages/compass-aggregations/src/components/add-stage/add-stage.tsx

```diff
@@ -9,33 +9,24 @@ import {
 } from '@mongodb-js/compass-components';
 import { PIPELINE_HELP_URI } from '../../constants';
 
-const iconContainerStyles = css({
-  textAlign: 'center',
-  marginTop: spacing[1] / 2,
-  marginBottom: spacing[1] / 2,
-});
-
-const buttonContainerStyles = css({
-  textAlign: 'center',
-  marginTop: spacing[4],
-  marginBottom: spacing[3],
```

Comment:
we have reduced spacing at the bottom of link now. we should consider increasing it to maintain the previous UX.